src/core_plugins/kibana/server/routes/api/export/index.js

```diff
@@ -0,0 +1,22 @@
+import exportDashboards from '../../../lib/export/export_dashboards';
+import Boom from 'boom';
+import moment from 'moment';
+export default function exportApi(server) {
+  server.route({
+    path: '/api/kibana/export/dashboards',
+    method: ['POST', 'GET'],
```

Comment:
Can we add validation to this endpoint? Not specifying a dashboard through the GET interface results in an exception which I feel should be caught at the router.